src/datepicker/types.js

```diff
@@ -257,25 +257,27 @@ export type StatefulDatepickerPropsT<T> = $Diff<
 
 export type TimePickerPropsT = {
   /** Render options in AM/PM format or 24 hour format. Defaults to 12 hour. */
-  format?: '12' | '24',
+  format: '12' | '24',
   /** Callback for when time selection changes. */
   onChange?: Date => mixed,
   overrides?: {
     Select?: OverrideT<*>,
   },
+  /** Set to true to allow times that aren't displayed in the options list to be entered manually. */
+  creatable: boolean,
   /** Amount of seconds between each option time. Defaults to 900 (15 minutes). */
-  step?: number,
```

Comment:
Please maintain the optional prop here and in `format`, and similarly make `creatable` optional. I understand that it is defined in default props, but when publishing an api to consumers, it should reflect what they should care about.

Reply:
👍 I changed these back to the original format, let me know if there are any additional changes I should make here